packages/core-plugin-api/src/plugin/Plugin.tsx

```diff
@@ -83,6 +83,10 @@ export class PluginImpl<
   toString() {
     return `plugin{${this.config.id}}`;
   }
+
+  getTranslationRef() {
```

Comment:
I'm thinking we'll want this to just be `translations`. At least the usage I'm envisioning right now is some form of reference to `catalogPlugin.translations` if you want to override or add translations for the catalog plugin from the outside.